code/routers/user/profile.js

```diff
@@ -55,7 +55,7 @@ router.get('/profile', (req, res) => {
           });
         }
         else{
-          userData['tripsCount'] = user[0].count;
+          userData['tripsCount'] = user[0] ? user[0].count : 0;
```

Comment:
This is already on master

Reply:
same as what I said above, but thanks for the hands up :)